sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -427,11 +425,10 @@ def test_binomial():
     for _ in (pi, -pi, nt, v, a):
         assert binomial(_, _) == 1
         assert binomial(_, _ - 1) == _
-    assert isinstance(binomial(u, u), binomial)
-    assert isinstance(binomial(u, u - 1), binomial)
-    assert isinstance(binomial(x, x), binomial)
-    assert isinstance(binomial(x, x - 1), binomial)
-
+    assert binomial(u, u) == 1
+    assert binomial(u, u - 1) == u
+    assert binomial(x, x) == 1
```

Comment:
The docstring should be clarified: the case of `k=-1` is superceded by the case of `n == k` which is the same as the case of `k==0`.

Reply:
about that clarification,
```python3
>>> u = Symbol('u', negative=True)
>>> binomial(u, u)
1
>>> binomial(-1, -1)
0
```
isn't this just wrong ? @smichr

Reply:
I'll update the docs... but we might have to correct for that discrepancy, because negative integers are a subset of negative numbers.